rulings/cards-rulings.yaml

```
@@ -3217,6 +3217,10 @@
   - His ability will make a diablerie unsuccessful (blood and equipment stay on him). [LSJ 20100228]
   - When he burns from aggravated damage, additional damage is lost. [ANK 20210424]
   - If {Amaranth} burns him and he comes back in torpor, another {Amaranth} can be played. [ANK 20210424]
+  - Yielding a contested Byzar does not trigger Byzar ability as he is (and his cardtext) not in play [LSJ 20100423]
+  - Byzar cardtext cannot be used when he is uncontrolled [LSJ 20100127]
+  - Cards that trigger when a vampire gets burned (e.g. {Soul Gem of Etrius}, trophy, etc) cannot be used when Byzar's ability is used [LSJ 20091217]
+  - Byzar ability can be used when he is burned to pay for an effect (e.g. burned to destroy an {Antediluvian Awakening}) [LSJ 20091217-2]
```

I think these should be grouped, with a proper global explanation... Also, ideally you want your "eg." to be the ones used by the rules director in the ruling you link, as much as possible.

```suggestion
  - His ability is a replacement effect. It prevents any effect triggering after the vampire burns (eg. {Sacrificial Lamb}), but still allows burn conditions to be met (eg. {Judgment: Camarilla Segregation}). [LSJ 20091217] [LSJ 20091217-2]
```